test/edu/csus/ecs/pc2/core/util/AbstractTestCase.java

```diff
@@ -1566,4 +1567,11 @@ public void assertNoPermssion(edu.csus.ecs.pc2.core.security.Permission.Type typ
         assertFalse("Expected for  " + account + " Permission " + type, account.isAllowed(type));
     }
     
+    public void ensureStaticLog() {
+        
+        if (StaticLog.getLog() == null) {
+            String testDirectory = getOutputDataDirectory();
```

Comment:
It doesn't seem right that the "static log" should be placed in the "output data" directory -- shouldn't it be in the "logs" directory, like it is in the method `ensureStaticLog()` in the `SampleContest` test?

And if so, then why are there two methods to "ensure static log"?  It seems like the one in the Sample Contest test should be omitted, and this one (in AbstractTestCase) should be used throughout...

Reply:
again this is out of scope.

Reply:
See my reply to your earlier "out of scope" comment.  The same applies here.

Reply:
pushed fix, now pc2 uses LogUtilities ensureStaticLog method